modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -92,6 +92,29 @@ function toggleFilesInCart(incomingFile) {
   };
 }
 
+function removeFilesFromCart(files) {
+  return (dispatch, getState) => {
+    const filesToRemove = _.isArray(files) ? files : [files];
+    const existingFiles = getState().cart.files;
+    const nextFiles = _.differenceBy(existingFiles, filesToRemove, 'file_id');
+    const filesRemoved = existingFiles.length - nextFiles.length;
+    dispatch(notify(getNotificationComponent('remove', `remove/${filesToRemove.file_name}`, {
```

Comment:
Thanks
Switched to using timestamp in the id
https://github.com/NCI-GDC/portal-ui/pull/1277/commits/490701336d00b4dc02cf6246797d94d209ace25e